lib/medtronic/processData.js

```diff
@@ -139,24 +165,39 @@ var init = function(config, settings) {
   cfg.settings = _.clone(settings);
 };
 
+var updateConfig = function(config) {
```

Comment:
I tracked this down earlier, and now seeing it's a new add... What's the motivation behind `_.clone`-ing the config in the driver in the first place? Mutation all over the place isn't necessarily my first design choice but if memory serves it's quite common in the device code - common enough that it stuck out at me when I saw you adding properties to `cfg` in the driver and *then* calling this update function.